roles/network_plugin/calico/templates/cni-calico.conflist.j2

```diff
@@ -28,6 +28,15 @@
         "subnet": "usePodCidr"
       },
 {% endif %}
+{% if calico_feature_control is defined %}
```

Comment:
I think setting `calico_feature_control: {}` will cause it to be defined? If so, you can maybe also check if the length as it's a list?

Reply:
Yep I totally agree :-)

Reply:
I changed the control of the list...